### PR TITLE
Add splitjoin.nvim

### PR DESCRIPTION
**Pull request by bennypowers**

### CODE REVIEW

**Review by phaazon — COMMENTED** [review]
Please target the right branch, not `master`. I’m changing the branch for you.

File: contents/2023/Feb/13/3-new-plugins/02-local-highlight.nvim.md [hunk 1]
```diff
@@ -11,15 +11,11 @@
...
-Multiple plugins to highlight the word under the cursor exist. However, none of them solves all of the following issues
-at once:
-
-1. Performance (especially on large files).
-2. Format mangling: do not mangle the current formatting options of the
-   highlighted words (e.g., italics, treesitter highlights).
-
-`local-highlight.nvim` will only process the currently visible parts of your buffer, and use ext-marks to highlight the 
-word under the cursor.
+A plugin highlighting the word under the cursor using ext-marks, similar in spirit to plugins like
```
Comment (phaazon) [inline contents/2023/Feb/13/3-new-plugins/02-local-highlight.nvim.md:14]: Unwanted changes due to bad target branch.

**Review by phaazon — CHANGES_REQUESTED** [review]

File: contents/2023/Feb/13/3-new-plugins/02-local-highlight.nvim.md (hunk 1 above)
Comment (bennypowers) [reply contents/2023/Feb/13/3-new-plugins/02-local-highlight.nvim.md]: checked out from main

Comment (phaazon) [reply contents/2023/Feb/13/3-new-plugins/02-local-highlight.nvim.md]: The easy way thing for you to do is to drop the commit and rebase on `2023-Feb-20`:

```sh
git fetch origin
git rebase -i origin/2023-Feb-20
# in the interactive editor, drop any commit that is not yours and fixup your commits
```

Comment (bennypowers) [reply contents/2023/Feb/13/3-new-plugins/02-local-highlight.nvim.md]: ok

**Review by phaazon — APPROVED** [review]
Thank you!

**Pull request merged**